dwave/plugins/sklearn/utilities.py

```diff
@@ -224,3 +234,267 @@ def dot_2d(a: npt.ArrayLike, b: npt.ArrayLike, *,
             out.flush()
 
     return out
+
+    
+def estimate_mi_matrix(    
```

Comment:
IMO this function makes more sense in the `transformers.py` module as it's rather specific to the transformer.